easybuild/tools/filetools.py

```diff
@@ -189,11 +188,19 @@ def is_readable(path):
         raise EasyBuildError("Failed to check whether %s is readable: %s", path, err)
 
 
+def _open(path, mode):
```

Comment:
Maybe the function name could be a bit better here, like `_smart_open_file` or `_safe_open_file`?

Reply:
Couldn't think of anything useful. It is a wrapper around `open`, so maybe `_open_wrapper`? I wanted to keep the name simple as it basically just does call `open` and using a consistent default encoding, which is even basically a no-op in most cases. Nothing smart here ;)

Reply:
I prefer `_open_wrapper` over just `_open` then, I can't think of a better name myself either. :)

Reply:
Maybe it should even be `open_wrapper`, so it's a public function that can be used in other places outside of the `filetools` module?

Reply:
Went with `open_file` which is similar to read_file, write_file. We could name it `open_text_file` which would be more correct and tell, that we don't need it for binary files. But then it might be confusing, so I just changed the docstring and added a comment